src/framework/vst/view/vstview.cpp

```diff
@@ -103,6 +107,10 @@ void VstView::init()
         return;
     }
 
+#ifdef Q_OS_WIN
+    FlickeringWorkaround::preventFlickering(window());
```

Comment:
What do you think about moving this to WindowsController?

Reply:
I don't think it's necessary, this is a workaround (I hope temporary) related with external native windows, therefore it's specific for VstView (we only have an external native window here)